tools/configurator/components/bios_check/bios_check.gd

```diff
@@ -41,4 +41,5 @@ func _ready():
 			for i in bios_line.size():
 				table_line.set_text(i, bios_line[i])
 				if table_line.get_index() % 2 == 1:
-					table_line.set_custom_bg_color(i,Color(1, 1, 1, 0.1),false)
+					table_line.set_custom_bg_color(i,Color(0.23, 0.54, 0, 1),false)
```

Comment:
I don't think this works with greyish brownish theme we are using right now
Could you make it a bit more neutral?